features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioBLE.cpp

```diff
@@ -510,7 +510,7 @@ void BLE::stack_setup()
 
     stack_handler_id = WsfOsSetNextHandler(&BLE::stack_handler);
 
-    HciSetMaxRxAclLen(100);
```

Comment:
that does look wrong, but the acl length is not the same as att_mtu, there's a acl buff length config somewhere

Reply:
Hi,
I had raised a Jira earlier regarding this issue earlier but did not see any update on this.
https://github.com/ARMmbed/mbed-os/issues/10315
This is an internal workaround we had to get Homekit working on mbed. Please suggest if there is a separate acl bufflength configuration somewhere which we can use. I will make the corresponding change.